packages/components/components/ConvertKit/ConvertKit.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { useEffect } from "react";
+
+const ConvertKit = ({ id }: { id: string }): React.ReactElement => {
+  useEffect(() => {
+    if (!document.getElementById(id)) {
+      const script = document.createElement("script");
+      script.async = true;
+      script.src = `https://prodigious-trader-7332.ck.page/${id}/index.js`;
+      script.id = id;
+      document.head.appendChild(script);
+    }
+  }, [id]);
```

Comment:
We need this useeffect instead of just embedding in the document because otherwise Next's hydration phase will just clear out the input